ccloud/docs/index.rst

```diff
@@ -428,13 +411,17 @@ Troubleshooting the demo
 1. If you can't run the demo due to error messages such as "'ccloud' is not found" or "'ccloud' is not initialized", validate that you have access to an initialized, working |ccloud| cluster and you have locally installed |ccloud| CLI.
 
 
-2. To view log files, look in the current Confluent CLI temp director (requires demo to be actively running):
+2. To view log files, look in the current Confluent CLI temp directory (requires demo to be actively running):
 
    .. sourcecode:: bash
 
-        $ ls `confluent current | tail -1`
+        # View all files
+        $ ls `confluent local current | tail -1`
```

Comment:
Not related to this exact change, but for your future consideration... 

The confluent local warning is written to STDERR, so you can obfuscate the warning without having to use `tail` or other search function with:
`confluent local current 2> /dev/null` 

This _might_ be more reliable in various situations, not sure about here

Reply:
@rspurgeon thanks for the suggestion, I filed a separate jira to track that